learning_resources/filters.py

```diff
@@ -53,23 +49,15 @@ def filter_offered_by(self, queryset, _, value):
         """OfferedBy Filter for learning resources"""
         return queryset.filter(offered_by__name__contains=OfferedBy[value].value)
 
-    def filter_audience(self, queryset, _, value):
-        """Audience filter for learning resources"""
-        if value == "professional":
-            queryset = queryset.filter(platform__audience=PROFESSIONAL)
-        else:
-            queryset = queryset.exclude(platform__audience=PROFESSIONAL)
-        return queryset
-
     def filter_platform(self, queryset, _, value):
         """Platform Filter for learning resources"""
         return queryset.filter(platform__platform=value)
 
     class Meta:
         model = LearningResource
         fields = [
-            "platform__audience",
-            "platform__platform",
+            "is_professional",
```

Comment:
I **think** that this should just be `professional`.

Currently `?professional=true` and `?is_professional=true` both work for filtering and both show up at http://localhost:8063/api/v1/schema/redoc/#tag/learning_resources/operation/learning_resources_list

**Or:** could just use `is_professional` as the name 🤷